mmv1/third_party/terraform/website/docs/r/dns_record_set.html.markdown

```diff
@@ -178,6 +178,69 @@ resource "google_dns_record_set" "geo" {
 }
 ```
 
+#### Primary-Backup
+
+```hcl
+resource "google_dns_record_set" "a" {
```

Comment:
While the tests are probably ok, I think we should have someone from the service team check out that this example actually makes sense for real-world usage, and possibly provide others.